.travis.yml

```
@@ -1,6 +1,6 @@
 # Test build the MoveIt tutorials. Author: Dave Coleman
 sudo: required
-dist: xenial
+dist: bionic
 language: ruby
 rvm:
   - 2.4
```

```suggestion
  - 2.5
```
This may fix it because it's failing with the following error
```
The last version of parallel (~> 1.3) to support your Ruby & RubyGems was 1.19.2. Try installing it with `gem install parallel -v 1.19.2` and then running the current command again
	parallel requires Ruby version >= 2.5. The current ruby version is 2.4.10.364.
```

---

I created a different PR (#556) that uses this fix and then the build works. Not sure if we should update this one or use #556.